test/dataframerow.jl

```diff
@@ -45,7 +45,7 @@ module TestDataFrameRow
     @test hash(DataFrameRow(df, 2)) != hash(DataFrameRow(df, 6))
 
     # check that hashrows() function generates the same hashes as DataFrameRow
-    df_rowhashes, _ = DataFrames.hashrows(df, false)
+    df_rowhashes, _ = DataFrames.hashrows(tuple(DataFrames.columns(df)...), false)
```

Comment:
I would rather write `Tuple(columns(df))` to avoid splatting and you do not have to quality `columns` with `DataFrames.`.